### ISSUE TITLE
something about reproduce the performance of the paper

### ISSUE THREAD

**Issue by zwl666666** [MichiganCOG/TASED-Net#9]
hello,
    thank you for your excellent work. I met some problems when trying to reproduce the paper. Whether it is the model you published or the model trained based on the code you gave，the NSS metric did not exceed 2.6. So I wonder if you have adopted some training tricks to make the result so good. The only difference is that batchsize uses 8 instead of 40 in the paper. I don’t know if this is the reason for the difference.
     look forward to your reply
     thank you

**Comment by kylemin** [comment]
Hi,

Thank you for your interest.
No, we did not use any training tricks. You can reproduce the performance if you just run the script `run_train.py`.
Usually, reducing the batchsize affects the performance a lot when training a deep neural network. In addition, optimal number of iterations might heavily depend on the batchsize.
I recommend you to increase the number of iterations if you have no choice but to reduce the batchsize.
For example, if you use batchsize 8, try [1500,1900,2000] rather than [750,950,1000] (or [1250,2000,2250]. It might be required to perform the experiments multiple times to find the better performance with the reduced batchsize).
Plus, please be sure to use images in PNG format and to resize the output saliency maps to the original size (640x360) when evaluating your output. Please do not resize the target (ground-truth) saliency maps.
Thank you.

**Comment by zwl666666** [comment]
I'm honored to receive your reply
I'll try it soon. And would you mind showing the result about predicting multiple saliency maps all at once for each sliding window？ Do you think it can improve its performance by finding a suitable decoder？
Thanks again for your work.

**Comment by kylemin** [comment]
You are welcome.
We performed some experiments of predicting multiple saliency maps at once (2 maps for each sliding window). However, we could not observe any outstanding performance gain. But yes, we believe that finding a suitable decoder can improve the performance in that case.
Thank you!

**Comment by zwl666666** [comment]
ok, thank you！looking forward to further communication with you in the future

**Comment by kylemin** [comment]
Sure, let's keep in touch.
